backend/api/v2beta1/job.proto

```diff
@@ -61,10 +61,18 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 
 
 service JobService {
-  // Creates a new job.
-  rpc CreateJob(CreateJobRequest) returns (Job) {
+  // Creates a new job in an experiment, given the experiment ID.
+  rpc CreateJobByExperiment(CreateJobByExperimentRequest) returns (Job) {
     option (google.api.http) = {
-      post: "/apis/v2beta1/jobs"
+      post: "/apis/v2beta1/experiments/{id}/jobs"
+      body: "job"
+    };
+  }
+
+  // Creates a new job in an experiment, given the experiment ID.
+  rpc CreateJobByExperimentName(CreateJobByExperimentNameRequest) returns (Job) {
+    option (google.api.http) = {
+      post: "/apis/v2beta1/experiments/names/{name}/jobs"
```

Comment:
`{name}` needs to comply with a field name in CreateJobByExperimentNameRequest

So, `{name}` needs to become `experiment_name` or via versa